### DISCUSSION TITLE
¿Cómo usar Kali en WSL?

### DISCUSSION THREAD

**Discussion by tapilew** [hackatomik/SISCOx256#1] [Preguntas Técnicas | Lab]
Texto de ejemplo

**Comment by FernanRixer** [comment]
Se usa como un entorno grafico que va por encima del WSL para poder similar una pc real... básicamente como una interfaz grafica

**Reply by JLCQ303** [reply]
hola

**Comment by tapilew** [comment]
Hola :)
